build.properties

```diff
@@ -8,17 +8,14 @@ dir.src=${project.basedir}/src/mg/${name}
 dir.vendor=${project.basedir}/vendor
 dir.vendor.bin=${dir.vendor}/bin
 dir.vendor.php=${dir.vendor}/php
-phing=${php.exec} ${dir.vendor.php}/phing.php -Dversion=${version}
+phing=${php.exec} ${dir.vendor}/phing/phing/bin/phing.php -Dversion=${version}
```

Comment:
This will probably also work:

`vendor/bin/phing -Dversion=${version}`

Binaries are usually installed in vendor/bin by composer.